### PR TITLE
Fix: cluster updateStatus.

### PR DESCRIPTION
**Pull request by lingdie**

### CODE REVIEW

File: pkg/types/v1beta1/cluster.go [hunk 1]
```diff
@@ -88,6 +93,26 @@ type ClusterCondition struct {
...
+		Message:           "Applied to cluster successfully",
+	}
+}
+
+func NewFailedClusterCondition(message string) ClusterCondition {
+	return ClusterCondition{
+		Type:              ClusterConditionTypeError,
+		Status:            v1.ConditionFalse,
+		LastHeartbeatTime: metav1.Now(),
+		Reason:            "Error",
```
Comment (cuisongliu) [inline pkg/types/v1beta1/cluster.go:111]: RunCluster

Comment (lingdie) [reply pkg/types/v1beta1/cluster.go:111]: reason？

File: pkg/apply/applydrivers/apply_drivers_default.go [hunk 2]
```diff
@@ -123,22 +122,15 @@ func (c *Applier) initStatus() {
...
-		Status:            v1.ConditionTrue,
-		LastHeartbeatTime: metav1.Now(),
-		Reason:            "Ready",
-		Message:           "Applied to cluster successfully",
-	}
-	c.ClusterDesired.Status.Phase = v2.ClusterSuccess
-	if clusterErr != nil {
-		condition.Status = v1.ConditionFalse
-		condition.Reason = "ApplyClusterError"
-		condition.Message = clusterErr.Error()
```
Comment (lingdie) [inline pkg/apply/applydrivers/apply_drivers_default.go:137]: should I change condition type if clusterErr is not nil?

Comment (cuisongliu) [reply pkg/apply/applydrivers/apply_drivers_default.go:137]: > should I change condition type if clusterErr is not nil?

yes.

File: pkg/types/v1beta1/cluster.go (hunk 1 above)
Comment (cuisongliu) [reply pkg/types/v1beta1/cluster.go:111]: > reason？

yes. error is unclear.

Comment (lingdie) [reply pkg/types/v1beta1/cluster.go]: ok.

**Review by cuisongliu — APPROVED** [review]

File: pkg/apply/applydrivers/apply_drivers_default.go (hunk 2 above)
Comment (lingdie) [reply pkg/apply/applydrivers/apply_drivers_default.go:137]: ok

**Pull request merged**